### ISSUE TITLE
Reject button does not refresh GUI

### ISSUE THREAD

**Issue by tsieger** [csn-le/wave_clus#24]
It seems that pressing the `Reject` button merges the corresponding cluster with the garbage spikes (i.e. the cluster 0) internally, but does not refresh the GUI: the cluster 0 seems to stay intact.

The evidence that `Reject`-ing merges internally can be demonstrated by the fact that forcing cluster assignment (using the `Force` button) and then going back (pressing the `Force` button once again) results in spikes of the rejected cluster being merged with cluster 0 correctly.

(To reproduce, e.g. use the `C_Easy1_noise01.mat` data having the `par.stdmin` set to 4. Cluster 4 emerges and it does not get merged with cluster 0 using the `Reject` button).

**Comment by ferchaure** [comment]
You are right, we don't update the cluster 0 after a reject.  But any other action that plot all the clusters again will plot correctly the new cluster 0 ( for example you can press "Plot all", and do only that).

We do this because a common way to use wave_clus is making a lot of overclustering and then rejecting clusters, and we didn't want to spend time plotting the cluster 0 for each rejection. Especially because, with real data, it's difficult to see if something is added to the cluster 0.
In any case I will think in a way to improve this.
